docs/source/maintenance/add_peer.rst

```diff
@@ -0,0 +1,61 @@
+============
+Adding Peers
+============
+
+In HL Iroha, you can add new peers to the network while it is running.
+This is done by using a special command, `AddPeer <../develop/api/commands.html#add-peer>`_.
+
+Requirements
+============
+
+**There should be a peer that:**
+
+— runs with a Genesis Block (initial block of the blockchain) identical to the one on the peers already in the network;
+
+— has a resolvable address;
+
+— has a peer keypair (Ed25519 with SHA-3)
+
+**The account that is sending the transaction adding a peer must have a `root permission <../develop/api/permissions.html#root>`_ in their role - this must be set in the genesis block.**
+
+Usage
+=====
+
+As described in `the API reference <../develop/api/commands.html#add-peer>`_ to use the command, you will only need:
+
+— a public key of the peer that you want to add to the network;
+
+— resolvable IP address of the peer
+
+Steps:
+
+1. Create a network with ``root`` permission set up in the genesis block assigned to a user;
```

Comment:
same as above (`can_add_peer`/`can_remove_peer`)

Reply:
There is probably something else why it was said to me it is required. Maybe Andrei will be able to help

Reply:
Fixed :)